.storybook/preview.js

```diff
@@ -1,5 +1,5 @@
 import ThemeProvider from "../src/providers";
-import { defaultTheme } from "../src/themes";
+import {light, dark, blue} from "../src/themes";
```

Comment:
We have different prettier settings, in mine these have a white space around the curly braces

Reply:
Fixed after eslint update